src/components/screens/HistoryScreen.tsx

```diff
@@ -45,36 +76,21 @@ export const HistoryScreen = () => {
       </Display>
 
       <Container>
-        <Text>
-          The below is the TESTNET account you have just created. The account
-          used for displaying the Home balances is another hardcoded MAINNET
-          account used just for testing the balances UI with its prices. You can
-          tap to copy the account to your clipboard.
-        </Text>
-
-        {isLoading && <Text>Loading...</Text>}
-
-        {error && <Text color={THEME.colors.status.error}>{error}</Text>}
-
-        {account && (
-          <>
-            <Text>{account.accountName}</Text>
-            <Text
-              onPress={() =>
-                Clipboard.setString(
-                  `TESTNET testing account:${account.publicKey}`,
-                )
-              }
-            >
-              {account.publicKey}
-            </Text>
-          </>
-        )}
-
         <ButtonContainer>
+          <Text>{account?.publicKey}</Text>
+
           <Button isFullWidth onPress={handleLogout}>
             <Text>Logout</Text>
           </Button>
+
+          <Button
+            isFullWidth
+            onPress={handleWipeData}
+            isLoading={isWiping}
+            destructive
+          >
+            <Text>Debug: Wipe All Data</Text>
```

Comment:
Could we remove the `Text` tags from the buttons on this screen and just use the string directly so they render the expected style?